### PR TITLE
fix(twig): make image optional in multilink card

### PR DESCRIPTION
**Pull request by Shashika6**
Fixes https://github.com/international-labour-organization/designsystem/issues/864

### Notes

* Make image optional in multilink card
* Add max-width to wide and fluid when image is available. 

### Screenshot

Screenshot 2024-08-23 at 19 40 25

### CODE REVIEW

**Comment by Shashika6** [comment]
@justintemps I noticed that you have used 

`  &__size__fluid {
    --max-width: 100% !important;
  }`
  
  in the card css which is overriding the size for the fluid to be 100% is this okay ?

**Comment by justintemps** [comment]
> @justintemps I noticed that you have used
> 
> ` &__size__fluid { --max-width: 100% !important; }`
> 
> in the card css which is overriding the size for the fluid to be 100% is this okay ?

@Shashika6 yea, that's what gives the card fluid width to fill it's container

**Comment by justintemps** [comment]
@inesdgomes could we get your eyes on this:

1. [Navigate here](https://deploy-preview-1177--ilo-ui-twig.netlify.app/?path=/story/components-cards-multi-link-card--default)
2. Click on the button to remove the image
Screenshot 2024-09-02 at 10 01 08
3. Check to make sure that it looks ok

File: packages/styles/scss/components/_multilinkcard.scss [hunk 1]
```diff
@@ -125,6 +125,14 @@
                 display: none;
               }
             }
+
+            &#{$self}--no-image {
+              --max-width: #{px-to-rem(920px)};
```
Comment (justintemps) [inline packages/styles/scss/components/_multilinkcard.scss:130]: @Shashika6 what is this for? Why should there being an image or not affect the width of the card? Also, it doesn't seem to have any effect.

Comment (Shashika6) [reply packages/styles/scss/components/_multilinkcard.scss:130]: When there is no image, the component has a max-width of 920px. 

There was a requirement by design.

**Comment by justintemps** [comment]
Hey @Shashika6 apologies, can you revert the change I asked for this morning? I discussed with @inesdgomes and we agreed that: max-width is different for the card when it doesn't have an image **but only when size is set to `wide`**. Otherwise, it's unaffected.

**Comment by Shashika6** [comment]
Reverted the change again @justintemps

**Pull request merged**